docs/getting-started/rulesets.md

```diff
@@ -40,6 +40,24 @@ While running it with this object, it will succeed:
 }
 ```
 
+By default, Spectral processes each rule on resolved document with all $refs resolved.
+If you would like to have an original input supplied to your rule, you can place `isResolved` property as follows:
+
+```yaml
+rules:
+  my-rule-name:
+    description: Tags must have a description.
+    given: $.tags[*]
+    severity: error
+    isResolved: false # note - if not specified or true, a resolved document will be given
```

Comment:
I'm sorry this is a switch and not a result flag. I missed it before but now it makes sense. Can we make it `resolved: true/false`?  😅